.github/workflows/pr-checklist.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow automatically adds a comment containing a reviewer checklist
+# when a new pull request is opened.
+name: Add a comment with reviewer checklist when PR opened
+on: pull_request
+
+jobs:
+  pr-checklist:
+    runs-on: ubuntu-latest
+    name: pr-checklist
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: 'Comment PR'
+        uses: actions/github-script@0.3.0
+        if: github.event_name == 'pull_request'
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            var msg = `# Instructions for code reviewer
+
+            Hello reviewer, thanks for taking the time to review this PR!
+
+            - Please use this checklist during your review, checking off items that you have verified are complete!
+            - Then, comment comment on the pull request with your review indicating where you have questions or changes need to be made before merging.
+            - Remember to review **every line of code** you’ve been asked to review, look at the context, make sure you’re improving code health, and compliment developers on good things that they do.
+            - PR reviews are a great way to learn, so feel free to share your tips and tricks. However for changes not required to make, please include \`nit:\` (for nitpicking) before making the suggestion. For example, \`nit:\` I prefer using a \`data.frame()\` instead of a \`matrix\` because...
+            - Engage with the developer when they respond to comments and check off additional boxes as they become complete so the PR can be merged in when all the tasks are fulfilled.
+
+            ## Checklist
+
+            - [ ] The code is well-designed.
+            - [ ] The functionality is good for the users of the code.
+            - [ ] Any User Interface changes are sensible and look good.
+            - [ ] Any parallel programming is done safely.
+            - [ ] The code isn’t more complex than it needs to be.
+            - [ ] The developer isn’t implementing things they might need in the future but don’t know they need now.
+            - [ ] Code has appropriate unit tests.
```

Comment:
`nit:` could cross-link here to the appropriate sections of the developer guide e.g. unit tests just in case it's unclear to a reviewer what we define as "appropriate unit tests"

Reply:
Thanks, I linked the testing section of the manual!